src/Apps/Artist/Routes/Shows/__test__/Shows.test.tsx

```diff
@@ -1,55 +1,75 @@
 import { ShowsRouteFragmentContainer as ShowsRoute } from "Apps/Artist/Routes/Shows"
-import { MockBoot, renderRelayTree } from "DevTools"
-import { ReactWrapper } from "enzyme"
+import { MockBoot } from "DevTools"
+import { mount, ReactWrapper } from "enzyme"
 import React from "react"
-import { graphql } from "react-relay"
 import { Breakpoint } from "Utils/Responsive"
 
-// import { ShowsFixture } from "Apps/__test__/Fixtures/Artist/Routes/ShowsFixture"
-
-import {
-  CVFixture,
-  showsConnection,
-} from "Apps/__test__/Fixtures/Artist/Routes/CV"
+import { ShowsFixture } from "Apps/__test__/Fixtures/Artist/Routes/ShowsFixture"
+import { RelayStubProvider } from "DevTools/RelayStubProvider"
 
 jest.unmock("react-relay")
 
 describe("Shows Route", () => {
   let wrapper: ReactWrapper
 
-  const getWrapper = async (breakpoint: Breakpoint = "xl") => {
-    return await renderRelayTree({
-      Component: ShowsRoute,
-      query: graphql`
-        query Shows_Test_Query($artistID: String!) {
-          viewer {
-            ...Shows_viewer
-          }
-        }
-      `,
-      mockResolvers: {
-        Viewer: () => ({ viewer: { artist_currentShows: CVFixture } }),
-        Artist: () => ({ artist: CVFixture }),
-        ShowConnection: () => showsConnection,
```

Comment:
As `viewer` is the top-level query here, if `Viewer` was mockable in the sense that the JSON data returned from it would result in the rest of the resolvers returning correct data this would be a bit more usable but in this case each little piece needs to be properly mocked out.